views/js/runner/plugins/navigation/review/navigator.js

```diff
@@ -301,10 +305,15 @@ define([
 
                 this.autoScroll();
 
+                this.setState('skipahead-enabled', this.config.skipaheadEnabled);
                 this.setState('prevents-unseen', this.config.preventsUnseen);
-                if (this.config.preventsUnseen) {
-                    // disables all unseen items to prevent the test taker has access to.
-                    this.controls.$tree.find(_selectors.unseen).addClass(_cssCls.disabled);
+                if (!this.config.skipaheadEnabled) {
+                    this.controls.$tree.find(_selectors.skippable).removeClass(_cssCls.skippable);
+
+                    if (this.config.preventsUnseen) {
```

Comment:
is it really needed? the `prevent-unseen` option already covers that. All you have to do is: `if (this.config.preventsUnseen && !options.reviewSkipahead) {`

Reply:
I'm not sure where to answer better about scopes and "similarity" to `prevents-unseen`, so would just comment this one.
First of all `skipahead` is not so similar to `prevents-unseen` as it may seem. `prevents-unseen` will allow to navigate to any item not yet seen. And `skipahead` should allow navigating to the items in sections which has `Enable Skipping Ahead` mark checked in test creator. So, for example, when seeing 4 sections with only 3 of them allowed to be skipped ahead: a) `prevents-unseen` will allow to go to any item of any section, while b) `skipahead` will allow to navigate to any item of only those sections who are allowed to be skipped ahead.

That's why, in general, I decided to not rely on the existing option, but to move it to separate one, even if the functionality partially covered by existing one.

Then, about forcing scope to `testPart`, this one was discussed with Mark/Tom. As `skipahead` is fully a, some kind of, "debug" feature it should give a little more convenience to work with review panel (and because most customers has scope of review panel set to `testSection` which almost fully cancel the new feature use - in this case the one needs this, he could only use `prevents-unseen`).

So, these are the main reasons, to move it all to a different feature, to allow this working separately. 
p.s.
Also, as said, `skipahead` should work in any case it is enabled, no matter what `prevents-unseen` is set to.

Reply:
I see your point. However, I disagree with the scope change. Even if it is a debug feature, it will break somehow the main behavior. What happen if the customer has set the scope to the whole test? suddenly it will be restricted to a `testPart`. Having customer specifics disturbing the main component is not acceptable.

Regarding the `skipahead` option, the AC of the ticket clearly mention as long as a section (so each items of the section) has the option, the user can reach any of the other items of this section. This is the purpose of the `prevent-unseen` too (in reverse). No need to specify which item can be reached. The feature is more to define from which item the "move-to-any" is allowed. Or did I misread the ticket?